synchronicity/type_stubs.py

```diff
@@ -242,29 +242,6 @@ def add_function(self, func, name, indentation_level=0):
             self.parts.append(self._get_function_source_with_overloads(func, name, indentation_level))
 
     def _get_translated_class_bases(self, cls):
-        # get __orig_bases__ (__bases__ with potential generic args) for any class
```

Comment:
Not needed anymore, since __orig_bases__ will now already be properly updated for the wrapped classes 🎉